exercises/concept/ellens-alien-game/.docs/instructions.md

```diff
@@ -0,0 +1,116 @@
+# Instructions
+
+Ellen is making a game where the player has to fight aliens.
+She has just learned about Object Oriented Programming (OOP) and is eager to take advantage of what this paradigm has to offer.
+
+To Ellen's delight, you have offered to help and she has given you the task of programming the aliens that the player has to fight.
+
+## 1. Create the Alien Class
+
+Define the Alien class so that it accepts an x and a y position, and puts them into `x` and `y` variables.
+Every alien starts off with a health of 3, so initialize the `health` variable too.
+
+```python
+>>> alien = Alien(2, 0)
+>>> alien.x
+2
+>>> alien.y
+0
+>>> alien.health
+3
+```
+
+Now, each alien should be able to keep track of it's own position and health.
+
+## 2. The `hit` Method
+
+Ellen would like the Alien class to have a `hit` method that decrements the health of an alien object when it is called.
```

Comment:
Keeping the first bit as it is (those are back-quotes around hit, not... er... quote-quotes - it's `hit`, not 'hit', because it's supposed to be in the code.  Then again, maybe I'm going overboard with the back-quotes.)

Changing the second bit to your suggestion.